README.md

```diff
@@ -71,6 +72,44 @@ Composite graph definitions can get long and cumbersome, so you can define Label
     vars.graphite_keys = ["{used,max}", "{currentThreadsBusy,currentThreadCount}"]
     vars.graphite_labels = ["Heap", "Threads"]
 
+### Area mode
+It is possible to fill the area below the graphed lines by adding the following to your Service definition.
+
+    vars.graphite.area_mode = "all" // default : "all"
+    vars.graphite.area_alpha = "0.1" // default : "0.1"
+
+To set it globally, customize the configuration file with :
+
+    graphite_area_mode = all
+    graphite_area_alpha = 0.1
+
+For details about the values, see doc here : http://graphite.readthedocs.io/en/latest/render_api.html?highlight=areaMode
+
+### Derivative graphs
+It is possible to plot derivative graphs instead of the measured values. This is useful for sensors giving increasing counters (like printer's page count, network IO bytes counter, ...). You can set it by adding the following to your Service definition.
+
+    vars.graphite.graph_type = "derivative" // default : "normal"
+
```

Comment:
Ok ...
This is a setting that should be set in Icinga's config.
I just thought it would be easier to set a variable to a single word, instead than to an expression (that could be miswritten).

In fact, the type of function can already be customized with the 2 variables : `summarize_interval` and `summarize_func` .

Do you see other ways to customize a derivative graph? Or you mean ... other than derivative?